# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../qdvdauthor/missheaders.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -ur qdvdauthor-0.1.2_orig/qdvdauthor/dialogcreatetemplate.cpp qdvdauthor-0.1.2/qdvdauthor/dialogcreatetemplate.cpp
--- qdvdauthor-0.1.2_orig/qdvdauthor/dialogcreatetemplate.cpp	2006-08-26 05:50:24.000000000 +0100
+++ qdvdauthor-0.1.2/qdvdauthor/dialogcreatetemplate.cpp	2006-09-09 20:35:02.000000000 +0100
@@ -8,6 +8,7 @@
 ****************************************************************************/
 #include <sys/types.h>
 #include <sys/stat.h>
+#include <stdlib.h>
 
 #include <qnetworkprotocol.h>
 #include <qradiobutton.h>
diff -ur qdvdauthor-0.1.2_orig/qdvdauthor/log.cpp qdvdauthor-0.1.2/qdvdauthor/log.cpp
--- qdvdauthor-0.1.2_orig/qdvdauthor/log.cpp	2006-08-26 05:50:24.000000000 +0100
+++ qdvdauthor-0.1.2/qdvdauthor/log.cpp	2006-09-09 20:35:02.000000000 +0100
@@ -4,6 +4,7 @@
 #include <qstring.h>
 #include <stdio.h>
 #include <time.h>
+#include <stdlib.h>
 
 #include "log.h"
 
diff -ur qdvdauthor-0.1.2_orig/qdvdauthor/templatedownload.cpp qdvdauthor-0.1.2/qdvdauthor/templatedownload.cpp
--- qdvdauthor-0.1.2_orig/qdvdauthor/templatedownload.cpp	2006-08-26 05:50:24.000000000 +0100
+++ qdvdauthor-0.1.2/qdvdauthor/templatedownload.cpp	2006-09-09 20:35:02.000000000 +0100
@@ -29,6 +29,7 @@
 
 #include <sys/types.h>
 #include <sys/stat.h>
+#include <stdlib.h>
 
 #include <qprogressbar.h>
 #include <qradiobutton.h>

